### ISSUE TITLE
CVL - opening images automatically

### ISSUE THREAD

**Issue by DrSpringy** [UQ-RCC/ipp#100]
I just tried to select 6 tif images they copied to cvl but FIJI didnt open as requested.

I then closed that instanced of CVL and tried to just open a single file, but I realised that the copy folder is dated today rather than by copy instance, and nothing was copied and FIJI didnt open

Do we need to create a new folder for each copy to scratch-CVL? and why is FIJI not opening?

**Comment by hoangnguyen177** [comment]
Fiji is not opening because the container image was moved. 
It is fixed now. 
Please test again. 

Regarding copying, it is only copied to a folder in scratch dated when the copy happens when data is not in scratch. Do you want a different behavior ?

**Comment by DrSpringy** [comment]
So I copied 3 files to CVL scratch and they opened in FIJI,
I then copied 3 other files from the same folder to CVL scratch and FIJI errored.
Is this because they're in the same CVL scratch folder as the last files, or something else?
Do they need to be in subfolders of todays date?

<img width="368" alt="Screen Shot 2021-11-18 at 11 17 34 am" src="https://user-images.githubusercontent.com/90365944/142333436-bd25a309-df9e-4a7a-a2f3-b6b373e010d4.png">

<img width="1586" alt="Screen Shot 2021-11-18 at 11 17 23 am" src="https://user-images.githubusercontent.com/90365944/142333425-20f0bcaa-0a95-4b1a-924c-a0842b99147e.png">
